cmd/gaia/cmd/gaiacli/main.go

```diff
@@ -68,13 +69,27 @@ func main() {
 	// the below functions and eliminate global vars, like we do
 	// with the cdc
 
+	// Module clients hold cli commnads (tx,query) and lcd routes
+	// TODO: Make the lcd command take a list of ModuleClient
+	mc := []sdk.ModuleClients{
+		govClient.NewModuleClient(storeGov, cdc),
+		distClient.NewModuleClient("", cdc),
```

Comment:
Why empty string? Do we never query the distribution store?

Reply:
There is currently no query functionality in the dist module. Rigel is working on that.

Reply:
We do already know the name of the store key though